src/components/network/NetworkParamToolbar.vue

```diff
@@ -1,19 +1,18 @@
 <template>
   <div class="networkParamToolbar" v-if="state.network">
-    <v-banner class="toolbar" height="40" max-height="40" single-line sticky>
+    <v-toolbar class="toolbar" dense flat height="40" max-height="40">
       <span class="d-flex">
         <v-btn-toggle
-          class="ma-2px"
           dense
           group
           mandatory
-          style="width: 100%"
           tile
           v-model="state.network.state.elementTypeIdx"
         >
           <v-btn
             :key="'elementType-' + item"
-            class="flex-grow-1"
+            class="flex-grow-1 ma-0"
+            height="40"
```

Comment:
Since this code is used multiple times - these options should be the same, right? Would it make sense to create an own class for this?

Reply:
I do not think so. `v-btn` provides a prop `height`. To me the code is fine.